_gsocproposals/2018/proposal_CERNBoxNewUI.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Petabyte-Scale Cloud Storage File Manager
+layout: gsoc_proposal
+project: CERNBox
+year: 2018
+organization: CERN, AARNet, ownCloud
```

Comment:
please add ownCloud and AARNet as "_gsocorgs/"

Reply:
Also, please specify organizations with the list syntax:
http://hepsoftwarefoundation.org/gsoc/guideline.html